### PR TITLE
Alias from File

### PR DESCRIPTION
**Pull request by nadizak**
400 char limit
alias and servalias

dooink

### CODE REVIEW

File: cogsmisc/customization.py [hunk 1]
```diff
@@ -145,6 +149,20 @@ def handle_alias_arguments(self, command, message):
...
+            return await self.bot.say('You must supply an alias name.')
+        if not 0 < len(ctx.message.attachments) < 2:
+            return await self.bot.say('You must attach a plain text file to use a file as the alias source.')
+        file = ctx.message.attachments[0]
+        async with aiohttp.get(file['url']) as f:
+          cmds = await f.read()
+          if len(cmds) > MAX_FILE_ALIAS_SIZE:
+              return await self.bot.say('Alias file upload is limited to {} characters.'.format(MAX_FILE_ALIAS_SIZE))
+        cmds = cmds.decode("utf-8")
+        return await ctx.invoke(self.bot.get_command("servalias" if servalias else "alias"), alias_name=alias_name, cmds=cmds)
```
Comment (mommothazaz123) [inline cogsmisc/customization.py]: This seems a little janky; why not make the routine just get the contents of the file, and set in the invoking command?

**Comment by nadizak** [comment]
It's all functional. Not sure on the alias truncation print. (Right now it's just a bunch of dots)

File: cogsmisc/customization.py (hunk 1 above)
Comment (nadizak) [reply cogsmisc/customization.py]: set in the invoking command?

Comment (nadizak) [reply cogsmisc/customization.py]: Ah okay i see what you mean. Yeah I can change it to that I guess. I missed a ( on the truncation string anyway so i'll do another commit later to fix those.

**Pull request closed without merging**